### DISCUSSION TITLE
Using a MIDI device to control a USB PTZ camera

### DISCUSSION THREAD

**Discussion by UUoocl** [WarmUpTill/SceneSwitcher#765] [Show and tell]
Advance Scene Switcher makes testing ideas so easy!  I wanted to control a camera with an Akai APC mini MIDI device.    
Advance Scene Switcher reads the slider position and sets the Move Transition plug-in filter.  

https://github.com/WarmUpTill/SceneSwitcher/assets/99063397/4987adee-61e9-4fb5-8da7-6a88f2d5c86e

![image](https://github.com/WarmUpTill/SceneSwitcher/assets/99063397/828defd5-cdc3-427c-ab94-c676aa5a4781)

Thanks for this awesome plug-in

**Comment by WarmUpTill** [comment]
That's really cool - thanks for sharing! :)
